Workbooks/WorkbooksMetadata.json

```diff
@@ -1429,5 +1429,18 @@
       "subtitle": "",
       "provider": "Microsoft",
       "featureFlag": "AzurePurview"
-    }   
+    },
+    {
+      "workbookKey": "ADXvsLA",
+      "logoFileName": "Azure_Sentinel.svg",
+      "description": "This workbook shows the tables from Azure Sentinel which are backed up in ADX. It also provides a comparison between the entries in the Azure Sentinel tables and the ADX tables. Lastly some general information about the queries and ingestion on ADX is shown.",
+      "dataTypesDependencies": [],
+      "dataConnectorsDependencies": [],
+      "previewImagesFileNames": [ "ADXvsLABlack.png", "ADXvsLAWhite.png"],
```

Comment:
Image extension does not match with the actual images that are added.
Image provided has extension as **PNG**  and the one used here is **png**

Please make it consistent at both the places and for both the preview images.

Validations are failing because of this

Reply:
changed in metadata.json